crates/bitwarden/src/mobile/tool/client_sends.rs

```diff
@@ -87,10 +87,8 @@ impl<'a> ClientSends<'a> {
     }
 }
 
-impl<'a> ClientVault<'a> {
-    pub fn sends(&'a self) -> ClientSends<'a> {
-        ClientSends {
-            client: self.client,
-        }
+impl<'a> Client {
```

Comment:
Are we not making a ClientTools sub-client for them, to match `vault` and `platform`?

Reply:
I'm a bit split, the tools team hace more well defined features than the other teams. And we currently expose generators and exporters directly on client.

If we decide to break out sends from tools this means we wouldn't have to refactor consumers code as much.